### PR TITLE
Moved "Cosmos DB"

### PR DESCRIPTION
**Pull request by yossi-y**
Moved "Cosmos DB" to Azure Diagnostics mode or resource-specific mode

### CODE REVIEW

**Comment by PRMerger20** [comment]
@yossi-y : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by bwren** [comment]
#sign-off

File: azure-monitor-ref/tables/azurediagnostics.md [hunk 1]
```diff
@@ -126,10 +125,11 @@ The following services use Azure diagnostics mode for their resource logs and se
 - Virtual network gateways
 - VPN Gateways
 
-### Azure Diagnostics mode or resource-specific mode
+###
```
Comment (ktoliver) [inline azure-monitor-ref/tables/azurediagnostics.md:128]: Hi @yossi-y - The PR description says that you move Cosmos DB to this section, but the section text has been deleted. Was that an oversight, or did you intend to remove the section H3? If the latter, we'll want to remove the H3 hashtag on this line.  (Either add heading text or remove the heading designator.) Thanks.

**Comment by ktoliver** [comment]
@yossi-y Thanks for fixing the header; merging now.

**Pull request merged**